setup.py

```diff
@@ -80,12 +80,13 @@ def get_version():
         'Framework :: Django :: 3.2',
         'Framework :: Django :: 4.0',
         'Framework :: Django :: 4.1',
+        'Framework :: Django :: 4.2',
```

Comment:
`Django :: 2.2` and `Django :: 3.0` apparently are not supported. Please, remove it.

Reply:
Remove